sebs/knative/triggers.py

```diff
@@ -0,0 +1,107 @@
+import concurrent.futures
+import datetime
+import json
+import subprocess
+from typing import Dict, List, Optional
+
+from sebs.faas.function import ExecutionResult, Trigger
+
+
+class KnativeLibraryTrigger(Trigger):
+    def __init__(self, fname: str, func_cmd: Optional[List[str]] = None):
+        super().__init__()
+        self.fname = fname
+        if func_cmd:
+            self._func_cmd = [*func_cmd, "invoke", "--target", "remote"]
+
+    @staticmethod
+    def trigger_type() -> "Trigger.TriggerType":
+        return Trigger.TriggerType.LIBRARY
+
+    @property
+    def func_cmd(self) -> List[str]:
+        assert self._func_cmd
+        return self._func_cmd
+
+    @func_cmd.setter
+    def func_cmd(self, func_cmd: List[str]):
+        self._func_cmd = [*func_cmd, "invoke", "--target", "remote"]
+
+    @staticmethod
+    def get_command(payload: dict) -> List[str]:
```

Comment:
Do we need this function? It looks like we could build the command directly on line 36 without the help of additional functions - more straightforward.